common/src/main/java/org/opensearch/ml/common/MLConfig.java

```diff
@@ -74,23 +106,42 @@ public void writeTo(StreamOutput out) throws IOException {
         }
         out.writeOptionalInstant(createTime);
         out.writeOptionalInstant(lastUpdateTime);
+        if (streamOutputVersion.onOrAfter(MINIMAL_SUPPORTED_VERSION_FOR_NEW_CONFIG_FIELDS)) {
+            out.writeOptionalString(configType);
+            if (mlConfiguration != null) {
+                out.writeBoolean(true);
+                mlConfiguration.writeTo(out);
+            } else {
+                out.writeBoolean(false);
+            }
+        }
+        out.writeOptionalInstant(lastUpdatedTime);
```

Comment:
Why not add this new field to if block above?

Reply:
sure, missed it somehow